kitsune/sumo/static/sumo/scss/base/forms/_fields.scss

```diff
@@ -355,6 +356,7 @@ textarea {
 }
 
 .delete-account-link {
+  float: right;
```

Comment:
This is the only change in the file.

Reply:
We can just change the text-align instead, and it needs to be wrapped in a `@include p.bidi()` to align left for rtl locales